### PR TITLE
doc: releases: add a note on debian packages & those built by the community

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/releases/octopus.rst [hunk 1]
```diff
@@ -23,12 +23,17 @@ General
...
+  - Debian Buster
   - :ref:`Container image <containers>` (based on CentOS 8)
 
   Note that the dashboard, prometheus, and restful manager modules
   will not work on the CentOS 7 build due to Python 3 module
   dependencies that are missing in CentOS 7.
 
+  Besides this packages built by the community will also available for the
+  following distros:
+  - Fedora 33
```
Comment (tchaikov) [inline doc/releases/octopus.rst]: you mean rawhive? as fedora33 is not yet released.

Comment (theanalyst) [reply doc/releases/octopus.rst]: got it, will change to rawhide

Comment (smithfarm) [reply doc/releases/octopus.rst]: neither is Leap 15.2, but you can already install it

**Comment by kalebskeithley** [comment]
To avoid ambiguity later on, e.g. after fedora 33 branches and rawhide becomes the staging for f34, I'd suggest that "fedora rawhide" be changed to fedora 33/rawhide or something similar.  At least that's what I do when I refer to rawhide.

**Comment by theanalyst** [comment]
> To avoid ambiguity later on, e.g. after fedora 33 branches and rawhide becomes the staging for f34, I'd suggest that "fedora rawhide" be changed to fedora 33/rawhide or something similar. At least that's what I do when I refer to rawhide.

Updated the release notes

File: doc/releases/octopus.rst [hunk 2]
```diff
@@ -23,12 +23,17 @@ General
...
+  - Debian Buster
   - :ref:`Container image <containers>` (based on CentOS 8)
 
   Note that the dashboard, prometheus, and restful manager modules
   will not work on the CentOS 7 build due to Python 3 module
   dependencies that are missing in CentOS 7.
 
+  Besides this packages built by the community will also available for the
+  following distros:
+  - Fedora (33/rawhide)
```
Comment (smithfarm) [inline doc/releases/octopus.rst]: Looks like elsewhere in this document `-` bullet lists are preceded by a blank line.

Do we need a blank line here, too?

Comment (theanalyst) [reply doc/releases/octopus.rst]: yes we do fixing

**Comment by tchaikov** [comment]
jenkins render docs

**Pull request merged**